src/main/java/org/broadinstitute/hellbender/engine/spark/GATKSparkTool.java

```diff
@@ -62,11 +62,6 @@
     @ArgumentCollection
     protected IntervalArgumentCollection intervalArgumentCollection = requiresIntervals() ? new RequiredIntervalArgumentCollection() : new OptionalIntervalArgumentCollection();
 
-    @Argument(doc = "maximum number of bytes to read from a file into each partition of reads. " +
-            "Setting this higher will result in fewer partitions. Note that this will not be equal to the size of the partition in memory",
-            fullName = "bamPartitionSize", shortName = "bps", optional = true)
-    protected long bamPartitionSplitSize = ReadsSparkSource.DEFAULT_SPLIT_SIZE;
-
```

Comment:
I think we still need to expose the input partition size as an argument here to allow the user to override the default, so that if and when we have memory problems with a tool (especially during development) we can tweak this knob and see if it helps. If we become confident that the Hadoop default is almost always right, then we can mark this as an "advanced" argument that should usually not be changed -- but I don't think we're there yet, and I think tools like BQSR are probably going to have memory issues with 128 MB partitions when run on large data.